hvantk/visualization/base.py

```
@@ -0,0 +1,191 @@
+        # Basic font size adjustments
+        plt.rcParams['font.size'] = 10 * font_scale
+        plt.rcParams['axes.labelsize'] = 11 * font_scale
+        plt.rcParams['axes.titlesize'] = 12 * font_scale
+        plt.rcParams['xtick.labelsize'] = 9 * font_scale
+        plt.rcParams['ytick.labelsize'] = 9 * font_scale
+
+
+def save_figure(fig: plt.Figure,
+                filename: str,
+                dpi: int = 300,
+                formats: list = ['png', 'pdf'],
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)